app/views/events/index.html.erb

```diff
@@ -8,66 +8,76 @@
   <center>
     <%= link_to 'New Event', new_event_path, class: "btn btn-success" %>
   </center>
+  <br>
 <% end %>
 
-<h4>Upcoming events</h4>
+<ul class="nav nav-tabs nav-justified">
+  <%= fancy_tag("Upcoming Events", events_path(tag: "UE") )%>
+  <%= fancy_tag("Past Events", events_path(tag: "PE") )%>
```

Comment:
```suggestion
  <li class="nav-item">
    <%= link_to("Upcoming Events", events_path(tag: "UE"), class: params[:tag] != "PE" ? 'nav-link active' : 'nav-link') %>
  </li>
  <li class="nav-item">
    <%= link_to("Past Events", events_path(tag: "PE"), class: params[:tag] == "PE" ? 'nav-link active' : 'nav-link') %>
  </li>
```

Instead of using the helper, check this in-place. We have two cases:
1) Params contains `tag = "PE"` then PE is active
2) Params does not contain anything (default) or contains `tag = "UE"` then UE is active

You would be able to scrap the helper method with this too. Let me know what you think

Reply:
I think this looks perfect, but the only reason I thought oh putting this logic in helper was to be able to reuse it when ever we need to do something similar.  But I know we shoudl should never code anticipating that something will be used in future so I guess I will follow your advice. Thanks a lot

Reply:
@amanpatel123 definitely can see the argument there. Unfortunately with having a default option (such as "UE") I'm not sure how we could make the code re-usable